src/stitching/errors.ts

```diff
@@ -52,8 +52,9 @@ export function annotateWithChildrenErrors(object: any, childrenErrors: Readonly
     ...object,
     [ERROR_SYMBOL]: childrenErrors.map(error => ({
       ...error,
-      ...(error.path ? { path: error.path.slice(1) } : {})
-    }))
+      ...(error.path ? { path: error.path.slice(1) } : {}),
+      originalError: error.originalError,
```

Comment:
This change fixed the originalError bubbling up for me, so I was able to log the original stack trace and format the error for the client appropriately with apollo-server.